public/locales/en/common.json

```diff
@@ -221,6 +221,11 @@
   "commerce>productPage>totalTokens": "Total tokens",
   "commerce>productPage>soldTokens": "Sold tokens",
   "commerce>productPage>description": "Description",
+  "shared>InfosKYC>errorRequest": "There was an error fetching the data, please try again later.",
   "auth>smartInputsController>subtitleInputFile": "Please include a selfie image in good lighting conditions and resolution",
-  "auth>formCompletKYCWithoutLayout>sendInforms": "Please wait, sending files"
+  "auth>ormCompletKYCWithoutLayout>saveInfosSucess": "Your information has been successfully saved!",
+  "auth>formCompletKYCWithoutLayout>sendInforms": "Please wait, sending files",
+  "shared>KYCStatys>approved": "Approved",
+  "shared>KYCStatys>denied": "Denied",
+  "auth>formCompletKYCWithoutLayout>notEditInfos": "It is not possible to edit the information as this user has already been reviewed"
```

Comment:
Your data has already been reviewed and approved. Please contact support if you need to update your information.